### PR TITLE
Ops-IAM Meeting minutes 20220922

### PR DESCRIPTION
**Pull request by o-otte**
Signed-off-by: Oliver Otte <otte@gonicus.de>

### CODE REVIEW

File: ops-iam/20220922.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+# 2022-09-22
+## Participants - use of github handle is preferred
+* @joshmue
+* @berendt
+* @fkr
+*@o-otte
```
Comment (fkr) [inline ops-iam/20220922.md:7]: please fix formatting

**Review by fkr — CHANGES_REQUESTED** [review]
see inline comment. thanks @o-otte!

File: ops-iam/20220922.md (hunk 1 above)
Comment (o-otte) [reply ops-iam/20220922.md]: Fixed with [a33d95e](https://github.com/SovereignCloudStack/minutes/pull/56/commits/a33d95ec2b4ef831041b3be04a82ffa6f38c1601)

**Review by fkr — APPROVED** [review]
awesome. thanks @o-otte!

**Pull request merged**